pkgs/default.nix

```diff
@@ -59,7 +59,15 @@ rec {
       inherit pkgs upgrade-maps;
     };
   };
-  inherit (bundle-locked) active-modules;
+
+  custom-bundle-locked = bundle-locked-fn {
+    modulesLocks = import ./filter-modules-locks {
+      inherit pkgs upgrade-maps;
+      moduleIds = [ "python-3.10" "nodejs-20" ];
```

Comment:
These lists of moduleIds for custom builds are starting to get out of sync between bundle-squasfs, custom-bundle-squashfs, and now custom-bundle-locked.